js/data/bucket/circle_bucket.js

```diff
@@ -44,7 +44,8 @@ CircleBucket.prototype.programInterfaces = {
                 return util.premultiply(layer.getPaintValue("circle-color", globalProperties, featureProperties));
             },
             multiplier: 255,
-            paintProperty: 'circle-color'
+            paintProperty: 'circle-color',
+            precision: 'lowp'
```

Comment:
Not presently. The pragmas are written like functions for future extensibility but are [replaced in the source per a lookup table](https://github.com/mapbox/mapbox-gl-js/blob/pragma/js/render/painter/use_program.js#L150-L151). To support this, we would need to make our pragma system more advanced or begin using glslify again.